### DISCUSSION TITLE
Image acquisition method

### DISCUSSION THREAD

**Discussion by DanielAsefa** [OpenStitching/stitching#124] [Q&A] [answered]
Dears,
I want to create a panoramic view of a wall structure that has various defects, including tiny cracks. Due to  capture the details, I need to take images up close to the structure (within a 0.3–1 meter range). When I positioned my tripod on one side near the target area and took a picture, it covered only a small portion of the scene. Is it possible to capture images by moving the tripod to capture the entire scene while maintaining a consistent distance from the structure, using the same focal length, and applying the same other settings?  Additionally, could you suggest an alternative method for capturing a series of images to cover the entire area?

With best Regards

**Comment by lukasalexanderweber** [comment]
You could be fine with that. Check out e.g. 

https://github.com/OpenStitching/stitching/discussions/81
https://github.com/OpenStitching/stitching/discussions/80
https://github.com/OpenStitching/stitching/discussions/84

for other use cases from similar distance

**Comment by DanielAsefa** [comment, marked as answer]
Thanks so much for @lukasalexanderweber 
Could you please provide resources on image acquisition methods for creating panorama images? The resource I found only covers capturing images from a single location.


with best regards!
